SPECIFICATION.md

```diff
@@ -0,0 +1,93 @@
+<h1>Provider Specification</h1>
+
+[TOC]
+
+# General
+- **name** Any record name supplied to the provider must be the full hostname.
+- **ttl** Reasonable default is 6 hours since it's supported by most services. Any service that does not support this must be explicitly mentioned somewhere.
+
+# API Operations
+## create_record
+- **Normal Behavior** Create a new DNS record.
```

Comment:
Should be changed to:
- **Normal Behavior** Create a new DNS record. Return a boolean `True` if successful.